simpletransformers/language_representation/representation_model.py

```diff
@@ -136,10 +136,14 @@ def encode_sentences(self, text_list, combine_strategy=None, batch_size=32):
         :param batch_size
         :return: list of lists of sentence embeddings(if `combine_strategy=None`) OR list of sentence embeddings(if `combine_strategy!=None`)
         """
+
+        self.model.to(self.device)
+
         batches = batch_iterable(text_list, batch_size=batch_size)
         embeddings = np.array([])
         for batch in batches:
```

Comment:
I think you should add here:
```
self.model.eval()
with torch.no_grad():
```
to save memory and computation.

Reply:
BTW I tested this PR in my code and it works very well. Thanks!

Reply:
You are right!